buildenv/jenkins/Jenkinsfile_personal

```diff
@@ -1,14 +1,15 @@
 #!groovy
+def JAVA_IMPL='hotspot'
 pipeline {
     agent {label "${params.MACHINE_LABEL}"}
     parameters {
     	string (defaultValue: "AdoptOpenJDK", description: 'personal repo', name: 'personalRepo')
     	string (defaultValue: "master", description: 'personal branch', name: 'personalBranch')
 		choice (choices: 'x64_linux\nx64_mac\ns390x_linux\nppc64le_linux\naarch64_linux', description: 'PLATFORM?', name: 'PLATFORM')
 		choice (choices: 'SE90\nSE80\nSE100', description: 'What is JAVA_VERSION?', name: 'JAVA_VERSION')
-		choice (choices: 'openjdk9-openj9\nopenjdk8-openj9\nopenjdk9\nopenjdk8\nopenjdk10\nopenjdk10-openj9', description: 'What is JVM Version?', name: 'JVM_VERSION')
+		choice (choices: 'openjdk9-openj9\nopenjdk8-openj9\nopenjdk9\nopenjdk8\nopenjdk10\nopenjdk10-openj9\nopenjdk10-sap', description: 'What is JVM Version?', name: 'JVM_VERSION')
 		string (defaultValue: '', description: 'Specific test project, leave blank by default for all projects to be compiled, e.g. openjdk_regression systemtest performance jck thirdparty_containers', name: 'TESTPROJECT')
-		string (defaultValue: 'runtest', description: 'Test Target to execute, default is runtest which will run all tests. You can also select sub-targets such as:\ntop level: openjdk, sanity, extended, perf, etc\ndirectory level: prefix a directory with test (eg test_systemtest)\nindividual test: jdk_bean_SE80_0, etc', name: 'TARGET')
+		string (defaultValue: 'runtest', description: 'Test Target to execute, default is runtest which will run all tests. You can also select sub-targets such as:\ntop level: openjdk, sanity, extended, system, jck, functional, perf, etc\ndirectory level: prefix a directory with test (eg test_systemtest)\nindividual test: jdk_bean_SE80_0, etc', name: 'TARGET')
```

Comment:
minor correction, should be jdk_bean**s**_SE80_0 (rather than jdk_bean_SE80_0)

Reply:
Updated. Thanks!